File: xamarin/Samples/Showcase-Android/Activities/CommentDisplayActivity.cs
using Android.App;
using Android.OS;
using Android.Widget;
using Com.Liferay.Mobile.Screens.Comment;
using Com.Liferay.Mobile.Screens.Comment.Display;

namespace ShowcaseAndroid
{
    [Activity]
    public class CommentDisplayActivity : Activity, ICommentDisplayListener
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.CommentDisplayView);

            CommentDisplayScreenlet commentDisplayScreenlet =
                (CommentDisplayScreenlet) FindViewById(Resource.Id.comment_display_screenlet);
            commentDisplayScreenlet.Listener = this;
        }

        /* ICommentDisplayListener */

        public void Error(Java.Lang.Exception p0, string p1)
        {
            System.Diagnostics.Debug.WriteLine($"Comment display error: {p0.Message}");
        }

        public void OnDeleteCommentSuccess(CommentEntry p0)
        {
            Toast.MakeText(this, "Comment delete success: " + p0.CommentId, ToastLength.Short).Show();
        }

        public void OnLoadCommentSuccess(CommentEntry p0)
        {
        	Toast.MakeText(this, "Comment load success: " + p0.CommentId, ToastLength.Short).Show();
        }

        public void OnUpdateCommentSuccess(CommentEntry p0)
        {
        	Toast.MakeText(this, "Comment update success: " + p0.CommentId, ToastLength.Short).Show();
        }
    }
}

File: android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/CommentDisplayActivity.java
package com.liferay.mobile.screens.testapp;

import android.os.Bundle;
import android.widget.CompoundButton;
import com.liferay.mobile.screens.comment.CommentEntry;
import com.liferay.mobile.screens.comment.display.CommentDisplayListener;
import com.liferay.mobile.screens.comment.display.CommentDisplayScreenlet;

/**
 * @author Sarai Díaz García
 */
public class CommentDisplayActivity extends ThemeActivity implements CommentDisplayListener {

    private CommentDisplayScreenlet commentDisplayScreenlet;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.comment_display);

        commentDisplayScreenlet = findViewById(R.id.comment_display_screenlet);
        commentDisplayScreenlet.setListener(this);
    }

    @Override
    public void onLoadCommentSuccess(CommentEntry commentEntry) {
        info(getString(R.string.comment_loaded_success, commentEntry.getCommentId()));
    }

    @Override
    public void onDeleteCommentSuccess(CommentEntry commentEntry) {
        info(getString(R.string.comment_deleted_success) + " " + commentEntry.getCommentId());
    }

    @Override
    public void onUpdateCommentSuccess(CommentEntry commentEntry) {
        info(getString(R.string.comment_updated_success) + " " + commentEntry.getCommentId());
    }

    @Override
    public void error(Exception e, String userAction) {
        error(getString(R.string.comment_error) + " " + userAction, e);
    }
}
